app/models/training_data/code_tags_sample.rb

```diff
@@ -18,6 +19,7 @@ class TrainingData::CodeTagsSample < ApplicationRecord
 
     self.exercise_id = solution.exercise_id unless exercise_id
     self.track_id = exercise.track_id unless track_id
+    self.llm_tags = tags unless llm_tags
```

Comment:
I thought that maybe this was the simplest way of doing this 🤷

Reply:
This isn't write, because the tags might not be coming from an LLM. I think just leave this out.

Reply:
```suggestion
```